ts-fold.el

```diff
@@ -167,10 +167,22 @@ the fold in a cons cell.  See `ts-fold-range-python' for an example."
   (let ((tree-sitter-mode t))
     (ts-fold-open-all)))
 
+(defvar ts-fold-close-keymap-default
+  (let ((map (make-sparse-keymap)))
+    (define-key map (kbd "TAB") 'ts-fold-open)
+    map))
```

Comment:
I think we can use `defvar-local` here? And get rid of the `make-local-variable` part.